internal/flink/command_compute_pool.go

```diff
@@ -12,12 +12,13 @@ import (
 )
 
 type computePoolOut struct {
-	IsCurrent bool   `human:"Current" serialized:"is_current"`
-	Id        string `human:"ID" serialized:"id"`
-	Name      string `human:"Name" serialized:"name"`
-	Cfu       int32  `human:"CFU" serialized:"cfu"`
-	Region    string `human:"Region" serialized:"region"`
-	Status    string `human:"Status" serialized:"status"`
+	IsCurrent  bool   `human:"Current" serialized:"is_current"`
+	Id         string `human:"ID" serialized:"id"`
+	Name       string `human:"Name" serialized:"name"`
+	CurrentCfu int32  `human:"Current CFU" serialized:"currrent_cfu"`
```

Comment:
This is a breaking change, so please update https://confluentinc.atlassian.net/wiki/spaces/DPI/pages/2907775380/Feature+Flag+Breaking+Changes

Reply:
updated